### ISSUE TITLE
grad-CAM

### ISSUE THREAD

**Issue by ebgoldstein** [ebgoldstein/Washover#8]
the implementation is [here](https://keras.io/examples/vision/grad_cam/)

need to visualize predictions from test set, then do grad-CAM overlays

**Comment by ebgoldstein** [comment]
https://github.com/ebgoldstein/MLWashover/commit/d8e1cec6ac13fd266ddf36a20f5a541b9745e387
